src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeaturesRecord.java

```diff
@@ -0,0 +1,121 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+        import htsjdk.samtools.util.Locatable;
+        import org.apache.avro.generic.GenericRecord;
+        import org.broadinstitute.hellbender.tools.variantdb.SchemaUtils;
+
+        import java.util.Objects;
+
+public class ExtractFeaturesRecord implements Locatable {
+
+    private final long location;
+    private final String contig;
+    private final int start;
+    private final int end;
+    private final String ref;
+    private final String allele;
+    private final Double rawQual;
+    private final Double refAD;                     // nullable
+    private final Float asMQRankSum;                // nullable
+//    private final String asMQRankSumFt; // not used?
+    private final Float asReadPosRankSum;           // nullable
+//    private final String asReadPosRankSumFt; not used?
+    private final Double rawMQ;
+    private final Double rawAD;
+    private final Double rawADGT1;
+    private final int sbRefPlus;
+    private final int sbRefMinus;
+    private final int sbAltPlus;
+    private final int sbAltMinus;
+    private final int numHetSamples;
+    private final int numHomvarSamples;
+
+    // FEATURE_EXTRACT_FIELDS = Arrays.asList(
+    //      LOCATION_FIELD_NAME,
+    //      REF_ALLELE_FIELD_NAME,
+    //      "allele",
+    //      RAW_QUAL,
+    //      "ref_ad",
+    //      AS_MQRankSum,
+    //      "AS_MQRankSum_ft",
```

Comment:
These are the frequency tables to be able to merge annotations... if they are in the SQL query, let's keep them here as well

Reply:
should I do the work of parsing the frequency tables into a usable format in java, or just leave them as strings? if the former, do you know which object type I should use?

Reply:
Just leave as a string for now

Reply:
oh thank goodness